.github/workflows/build.yml

```diff
@@ -63,3 +63,5 @@ jobs:
     - run: yarn --frozen-lockfile --ignore-scripts
 
     - run: yarn lint
+
+      - run: yarn format
```

Comment:
running prettier here to format the code will update the source code on the build machine. I don't think it will fail the build. Nor will it check-in the changes (Don't create an auto-commit). So, I don't think we should run `yarn format` here.

I'm not sure how you intended to use prettier. Should/Will `yarn lint` fail if it is not according to prettier settings? Please let me know.

Reply:
> > I'm not sure how you intended to use prettier.
> 
> I am taking over this task from Estelle so I had to do a little bit digging. I checked #861 and in the issue Rodrigo says that Prettier should be run to check the format of the code in CI. From what I understand, we were supposed to:
> 
> [1] Enable Prettier in the project so everyone can use it. We can set up a [pre-commit hook](https://prettier.io/docs/en/precommit.html) so that Prettier is ran every time the developer makes a commit. With this we ensure that the code is well-formatted before it get pushed to GitHub.
> 
Having prettier installed for users to run sounds good to me. I'm not sure to run the prettier script automatically with a pre-commit hook. Files get staged and it might get confusing. User could choose to add a pre-commit hook script in their repo if they wish, but that's on them. Note, if we realize later that the pre-commit hook should be there, we can add it later on.

> [2] Enable Prettier in CI to ensure that the code is well-formatted before merging. So we need to run `prettier --check` in CI as documented [here](https://prettier.io/docs/en/install.html).
> 
Updating the CI to add the check and fail the build makes sense. User's can then update the patch to run prettier.

> If this workflow makes sense, let me know and I can proceed with the changes.
> 
> > Should/Will `yarn lint` fail if it is not according to prettier settings?
> 
> According to Prettier documentation, there might be cases where Prettier might have a conflict with the styling config of linters. Most of the time, Prettier should not be causing any conflicts.
> 
> I think currently using ESLint as the linter for this repo (please correct me if I am wrong). Worst case, there is a version of Prettier that plays well with ESLint called [`eslint-config-prettier`](https://prettier.io/docs/en/integrating-with-linters.html). This should prevent any unexpected conflict between the two plugins. However, this means that we will be running Prettier as ESLint.
Thanks for the info.